src/components/find-on-shelf/getWayfinder.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * The 'getWayfinder` function sends an HTTP GET request to the Wayfinder API to get position data (holdings).
+ *
+ * @param {HoldingDataInterface} holdingsIds - An object containing the parameters for the request.
+ */
+
+import lodash from "lodash";
+import querystring from "querystring";
+import HttpError from "../../core/utils/errors/HttpError";
+import {
+  HoldingDataInterface,
+  WayfinderReaponse
+} from "../wayfinder/wayfinder-types";
+import {
+  getServiceBaseUrl,
+  serviceUrlKeys
+} from "../../core/utils/reduxMiddleware/extractServiceBaseUrls";
+
+export const easyWayUrl = "https://wf-dev.stg.easyscreen.io";
```

Comment:
Remove hardcoded wayfinder url. In other places change the name of such variable to wayfinder for consitency.